packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -312,6 +308,15 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
     function speedUpRelay(DepositData memory _depositData) public {
         bytes32 depositHash = _getDepositHash(_depositData);
         RelayData storage relay = relays[depositHash];
+        require(
```

Comment:
This adds 300k gas to this function. The total gas cost to slow+instant relay is roughly the same as before.

Before:
- relayDeposit = 1.2mil, speedupRelay = 0.01mil

After:
- relayDeposit = 1mil, speedupRelay = 0.3 mil